packages/snaps-jest/src/internals/simulation/interface.ts

```diff
@@ -154,43 +151,69 @@ function* getStoredInterface(
   return { ...payload, content };
 }
 
+/**
+ * Find an element inside a form element in a JSX tree.
+ *
+ * @param form - The form element.
+ * @param name - The element name.
+ * @returns An object containing the element and the form name if it's contained
+ * in a form, otherwise undefined.
+ */
+function getFormElement(form: FormElement, name: string) {
+  const element = walkJsx(form, (childElement) => {
+    if (
+      (childElement.type === 'Input' || childElement.type === 'Button') &&
```

Comment:
Is there a way to not hardcode these types like this?

Reply:
Maybe it could be extracted from `FormStruct.children`? 🤔

Reply:
I ended up removing this check completely, and just checking if the name matches. The function this is used in check for the proper type anyway. I've updated the error messages to be a bit more informative too.